### DISCUSSION TITLE
Make latest tag in npm registry point to the latest version

### DISCUSSION THREAD

**Discussion by Ph0tonic** [advanced-cropper/vue-advanced-cropper#223] [Q&A] [answered]
Hi,

I use yarn as my package manager and it relies on tags to identify the latest version when updating the version. Therefore it recommends using an older version.

However, the `latest` tag in the npm registry points to the latest version for the vue 2 version (1.x).

I think it would be nice to change this setting to point to the latest version (2.x) as vue 3 is now well-established.

Best,

**Comment by Norserium** [comment, marked as answer]
@Ph0tonic, I set the `latest` tag for `2.x.x` versions.

The versions schema looks now in the following way:
`vue-advanced-cropper@latest`: `2.x.x`
`vue-advanced-cropper@vue-3`: `2.x.x`
`vue-advanced-cropper@vue-2`: `1.x.x`

Also, I've updated the documentation in the corresponding way.

**Reply by Ph0tonic** [reply]
Great, thank you 👍

**Comment by Ph0tonic** [comment]
@Norserium We can resolve this discussion 👍

**Reply by Norserium** [reply]
It's up to you. I assume I can't resolve the discussion.

**Reply by Ph0tonic** [reply]
Hum, me neither, I haven't any button available to do so.

**Reply by Ph0tonic** [reply]
Ok, so I had to change the category to be able to mark it as answered, thanks!

**Reply by Norserium** [reply]
It's great. You are welcome!
